### PR TITLE
New blog post re: FundOSS & OpenCollective

### PR DESCRIPTION
**Pull request by zenhack**
This is a first draft of a blog post promoting the FundOSS drive and the community opencollective more broadly. I'm sure it could use some work. Thanks to @garrison for prodding us on this.

cc: @ocdtrekkie, @kentonv

### CODE REVIEW

File: _posts/2021-06-17-sandstorm-community-fundraising.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+Hey Everyone!
+
+Members of the Sandstorm community are doing some fundraising!
+
+Specifically,
+
+- In the immediate term, we're [participating in FundOSS][1], a donation
+  matching program that is a joint effort between [OpenCollective[2] and
```
Comment (ocdtrekkie) [inline _posts/2021-06-17-sandstorm-community-fundraising.md]: Looks like you are missing a ]

File: _posts/2021-06-17-sandstorm-community-fundraising.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+Hey Everyone!
+
+Members of the Sandstorm community are doing some fundraising!
+
+Specifically,
+
+- In the immediate term, we're [participating in FundOSS][1], a donation
+  matching program that is a joint effort between [OpenCollective][2]
```
Comment (zenhack) [inline _posts/2021-06-17-sandstorm-community-fundraising.md:15]: I'm a little fuzzy on exactly which orgs are involved in putting this together in the first place, @ocdtrekkie can you comment on whether I should add to/amend this?

**Review by ocdtrekkie — CHANGES_REQUESTED** [review]

File: _posts/2021-06-17-sandstorm-community-fundraising.md (hunk 2 above)
Comment (ocdtrekkie) [reply _posts/2021-06-17-sandstorm-community-fundraising.md:15]: Open Source Collective is the org partnering with GitCoin. I think it's operated by the OpenCollective folks, but technically OC is the platform and OSC is the fiscal host which is participating in creating FundOSS.

File: _posts/2021-06-17-sandstorm-community-fundraising.md (hunk 1 above)
Comment (zenhack) [reply _posts/2021-06-17-sandstorm-community-fundraising.md]: That was fast! I noticed this basically right after pushing and then force-pushed -- but you were already reviewing!

**Review by ocdtrekkie — APPROVED** [review]

**Pull request merged**